src/scss/utilities/mixins/typography/_soe-sans-mixins.scss

```diff
@@ -0,0 +1,46 @@
+
+$soe-font--sans: Source Sans Pro;
+
+@mixin soe-sans {
+  font-family: $soe-font--sans;
+}
+
+
+// Note that main difference with basic styles is font-weight
+@mixin soe-type-sans-base {
+  font-family: $soe-font--sans; // should already be, but we can make sure
+  font-weight: 900;
+  //color: $su-color-black; // kerri do we need this?
```

Comment:
@kerri-augenstein

Reply:
Pinging Kerri in slack about this question.

Reply:
Resolved, removed.